### ISSUE TITLE
🧹 : Move Loaders to showcase folder

### ISSUE THREAD

**Issue by Ansub** [Ansub/SyntaxUI#83]
### Description 📝

all of the loaders needs to be moved to `showcase` folder like the rest of components.

### Link 🔗

.

### Steps to Reproduce 🔄

.

### Screenshots 📸

.

### Expected Behavior 🤔

.

### Actual Behavior 😱

.

### Environment 🌍

.

**Comment by epoll31** [comment]
@Ansub I can work on this. Can I move them to separate files? Or, should I keep them together?

**Comment by Ansub** [comment]
yeah, they will go in a seperate files now, `showcase/components/loaders/Loader.tsx`
